tests/mocks.py

```diff
@@ -478,6 +477,31 @@ def transaction(
         )
         return transaction
 
+    def comment(self, content, author=None):
+        """Return a Phabricator Comment object.
+
+        Args:
+            content: The raw (Remarkup) comment contents.
+            author: Optional Phabricator User that authored the comment. If None
+                then a new User will be generated.
+        """
+        phid = self._new_phid("XCMT-")
```

Comment:
It's very strange that comments have PHIDs, but can't be found with a `phid.query`... wonder if that's considered an upstream bug or is intentional?